pkg/engine/template_generator_test.go

```diff
@@ -74,14 +75,36 @@ func TestGetTemplateFuncMap(t *testing.T) {
 		"GetVNETSubnets",
 		"GetDataDisks",
 		"HasBootstrap",
+		"GetMasterAllowedSizes",
+		"GetDefaultVNETCIDR",
+		"GetKubernetesAllowedVMSKUs",
+		"GetSizeMap",
+		"WriteLinkedTemplatesForExtensions",
+		"GetSshPublicKeysPowerShell",
```

Comment:
nit: could we rename this to `GetSSHPublicKeysPowerShell`

Reply:
Doing that would mean we'd have to update the Powershell script that consumes this template func. Generally I'm trying not to change any of the template func names or arity themselves if possible, to minimize change.

Reply:
sure. It's a one line change, I can change it in a separate PR once this merges.